bulk/sensor_bulk_load-AssetRecord.csv

```diff
@@ -1608,37 +1608,44 @@ CGINS-METBPR-00263,A01264,Sensor,0,Bulk Meteorology Instrument Package: METBK BP
-CGINS-METCTD-11116,A00183,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-11116,,20150604,5816,,
-CGINS-METCTD-11117,A00600,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-11117,,20140108,5816,original ooi barcode A000600,
-CGINS-METCTD-11123,A00601,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-11123E,,20140108,5816,,
-CGINS-METCTD-11558,A00716,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-11558,,20140227,5816,,
-CGINS-METCTD-11559,A00751,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-11559,,20140312,5816,,
-CGINS-METCTD-11560,A00752,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-11560,,20140312,5816,,
-CGINS-METCTD-11561,A00715,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-11561,,20140227,5816,,
-CGINS-METCTD-12754,A01266,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-12754,,20141214,5816,,
-CGINS-METCTD-12755,A01267,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-12755,,20141214,5816,,
-CGINS-METCTD-12756,A01268,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-12756,,20141214,5816,,
-CGINS-METCTD-12757,A01269,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-12757,,20141214,5816,,
-CGINS-METCTD-12758,A01270,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-12758,,20141214,5816,,
+CGINS-METCTD-08028,A00207,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37SM61212-8028,,20101122,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-10137,A00190,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-10137,,20130131,5816,Missing from inventory,Woods Hole Oceanographic Institution
+CGINS-METCTD-10138,A00197,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-10138,,20121226,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-10139,A00204,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-10139,,20130131,5816,Missing from inventory,Woods Hole Oceanographic Institution
```

Comment:
For 10137 and 10139, change Comment from "Missing from inventory" to "Lost at sea on CP01CNSM-00001".

Reply:
Done